docs/reporters/plain-text-templates.md

```diff
@@ -2,7 +2,7 @@
 
 The [`PlainTextTemplateReporter`](../../reporter/src/main/kotlin/reporters/freemarker/PlainTextTemplateReporter.kt)
 enables customization of the generated open source notices with [Apache Freemarker](https://freemarker.apache.org/)
-templates and producing any other arbitrary plain text files.
+templates and producing any other arbitrary plain text files, such as `.adoc` files.
```

Comment:
I was missing these changes when reviewing the first commit. Please squash to that commit. That also avoids the `fix`-type in the commit title, which is a bit awkward, as you're "fixing" something that you "broke" in the same PR.

Reply:
And in case you're going to ask whether the completion-script re-generation should then be squashed, too: No, I regard that as a different thing, as it's committing generated files vs. hand-made changes.